lib/starting_page.dart

```diff
@@ -40,15 +41,28 @@ class _StartingPageState extends State<StartingPage> {
             return Column(
               mainAxisAlignment: MainAxisAlignment.spaceEvenly,
               children: <Widget>[
-                customContainer(
-                  strings.statuePrefab,
-                  size,
-                  snapshot.data == strings.statuePrefab,
-                ),
-                customContainer(
-                  strings.cubePrefab,
-                  size,
-                  snapshot.data == strings.cubePrefab,
+                Expanded(
+                  child: GridView.count(
+                    crossAxisCount: 2,
+                    children: <Widget>[
+                      Padding(
```

Comment:
Instead of the `Padding` widget, I'd suggest adding `crossAxisSpacing` and `mainAxisSpacing` to the `GridView`, so as to eliminate adding the `Padding` widget every time we are adding a widget.